src/ports/postgres/modules/svm/svm.py_in

```diff
@@ -440,55 +595,68 @@ def _process_epsilon(is_svc, args):
 def _extract_params(schema_madlib, params, module='SVM'):
     # NOTICE: the type of values in params_default should be consistent with
     # the types specified in params_types
-    params_default = {'init_stepsize': 0.01,
-                      'decay_factor': 0.9,
-                      'max_iter': 100,
-                      'tolerance': 1e-10,
-                      'lambda': 1.0,
-                      'norm': 'L2',
-                      'n_folds': 0,
-                      'epsilon': 0.01,
-                      'eps_table': ''}
-
-    params_types = {'init_stepsize': float,
-                    'decay_factor': float,
-                    'max_iter': int,
-                    'tolerance': float,
-                    'lambda': list,
-                    'norm': str,
-                    'n_folds': int,
-                    'epsilon': float,
-                    'eps_table': str}
+    params_default = {
```

Comment:
Early stopping in optimization serves the similar purposes as regularizations do. Specifying those optimization parameters as lists allows users to CV over them so that they are chosen to provide the `best` generalization ability while avoiding over- or under-fitting. Another practical concern is to make it easier for users to choose a proper init_stepsize which varies from case to case depending on both the model and the data.